commands/commands.go

```diff
@@ -36,7 +36,7 @@ var suggestedBuilders = [][]suggestedBuilder{
 		{"Cloud Foundry", "cloudfoundry/cnb:cflinuxfs3", "larger base image with Java, Node.js & Python"},
 	},
 	{
-		{"Heroku", "heroku/buildpacks", "heroku-18 base image with official Heroku buildpacks"},
+		{"Heroku", "heroku/buildpacks:18", "heroku-18 base image with buildpacks for Ruby, Java, & more"},
```

Comment:
should we just list out all 6 languages?

Reply:
I you add a description to your builder config this hardcoded message will be irrelevant. I think we are hoping remove these hardcoded strings eventually. https://buildpacks.io/docs/using-pack/working-with-builders/#builder-configuration